config/config.go

```diff
@@ -787,6 +789,7 @@ func (c *Config) ToOptions() skipper.Options {
 		WhitelistedHealthCheckCIDR:             whitelistCIDRS,
 		KubernetesPathMode:                     c.KubernetesPathMode,
 		KubernetesNamespace:                    c.KubernetesNamespace,
+		KubernetesEnableEndpointslices:         c.KubernetesEnableEndpointSlices,
```

Comment:
We also need to update KubernetesDataClientOptions() to have this flag in routesrv.

Reply:
this was already done https://github.com/zalando/skipper/pull/2565/files#diff-9b0b16d06f27b640eb77f6ba0dee84df6a660528e5cccd81839e0330c4b92dd2R923